mytoyota/models/dashboard.py

```diff
@@ -28,74 +28,73 @@ def __init__(
     @property
     def legacy(self) -> bool:
         """If the car uses the legacy endpoints."""
-        if "Fuel" in self._vehicle.odometer:
-            return True
-        return False
+        return "Fuel" in self._vehicle.odometer
```

Comment:
Be aware that the `fuel` only exists if the car supports it

Reply:
This will only return a `True` if `"Fuel"` exists in the odometer. Otherwise a `False` will be returned. So it does nothing different from the previous code. Only a little simpler written ;-)

![4523524nt](https://user-images.githubusercontent.com/7945681/223949760-9faca2a6-a03e-49c5-a317-e26419e1b563.PNG)